core/misc/backdrop.js

```diff
@@ -760,7 +760,7 @@ Backdrop.featureDetect.flexbox = function() {
  * Example use:
  * @code
  *   Backdrop.optimizedResize.add(function() {
- *     console.log('Smooth AND resource effecient!');
+ *     console.log('Smooth AND resource efficient!');
```

Comment:
```suggestion
 *     console.log('Smooth AND resource-efficient!');
```

Here the noun is used as a modifier, so a hyphen would be appropriate.